spec/controllers/books_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+require 'helpers/shared_contexts/authenticated_user_spec'
+
+describe BooksController, type: :controller do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    context 'When fetching all the books' do
```

Comment:
I think you forgot to add a spec that checks that this endpoint is paginated. Here's again the link with the documentation https://github.com/Wolox/wor-paginate#test-helpers